zc_install/sql/install/mysql_zencart.sql

```diff
@@ -717,6 +717,19 @@ CREATE TABLE customer_groups (
   UNIQUE KEY idx_groupname_zen (group_name)
 );
 
+# --------------------------------------------------------
+#
+# Table structure for table 'customer_password_reset_tokens'
+#
+
+DROP TABLE IF EXISTS customer_password_reset_tokens;
+CREATE TABLE customer_password_reset_tokens (
```

Comment:
Instead of `DROP` and `CREATE`, how about just `CREATE IF NOT EXISTS` on the table (in case there are early adopters)?

Reply:
Fair question.

I opted for DROP/CREATE because that's the standard we use in the rest of the file.
And tokens are only valid for 1 hour, so the impact is essentially nil. Even during an upgrade.